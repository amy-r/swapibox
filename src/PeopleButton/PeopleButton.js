import React from 'react';
import propTypes from 'prop-types';

const PeopleButton = (props) => {


    return (
      <button onClick= {props.onClick} className= 'PeopleButton'>People</button>
    )
  
}

export default PeopleButton;