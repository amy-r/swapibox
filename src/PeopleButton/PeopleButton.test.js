 /* eslint-disable */ 
 
import React from 'react';
import ReactDOM from 'react-dom';
import PeopleButton from './PeopleButton';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

describe( 'People Button', () => {

  it('exists', () => {
  })

  it('renders without crashing', () => {
    const renderedComponent = shallow(<PeopleButton />, {disableLifecycleMethods: true})
    expect(renderedComponent).toMatchSnapshot();
  })

});