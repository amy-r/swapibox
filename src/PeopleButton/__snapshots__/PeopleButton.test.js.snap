// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`People Button renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PeopleButton />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "PEOPLE",
      "className": "PeopleButton",
      "onClick": undefined,
    },
    "ref": null,
    "rendered": "PEOPLE",
    "type": "button",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "PEOPLE",
        "className": "PeopleButton",
        "onClick": undefined,
      },
      "ref": null,
      "rendered": "PEOPLE",
      "type": "button",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
